mne/tests/test_epochs.py

```diff
@@ -905,7 +905,7 @@ def test_crop():
     assert_array_equal(data3, data_normal[:, :, tmask])
 
     # test time info is correct
-    epochs = EpochsArray(np.zeros((1, 1, 1000)), create_info(1, 1000., 'eeg'),
+    epochs = EpochsArray(np.zeros((1, 1, 1001)), create_info(1, 1000., 'eeg'),
```

Comment:
then it's not the right fix. What if I have 1000 samples and I want to call epochs.crop(-.200, .700) ?

it should work all the time.